apps/widget/package.json

```
@@ -53,9 +53,10 @@
     "start-server-and-test": "1.11.6"
   },
   "scripts": {
-    "start": "craco start",
-    "start:dev": "PORT=4500 pnpm run start",
+    "start": "PORT=4500 craco run start",
```

@djabarovgeorge I think this just should be `cracto start` 
```suggestion
    "start": "PORT=4500 craco start",
```